cf/commands/organization/orgs.go

```diff
@@ -51,6 +53,9 @@ func (cmd ListOrgs) Run(c *cli.Context) {
 	table := cmd.ui.Table([]string{T("name")})
 
 	orgs, apiErr := cmd.orgRepo.ListOrgs()
+
+	sort.Sort(models.Organizations(orgs))
```

Comment:
You should now add a test that verifies that listing organizations is indeed sorted correctly (even on a list of fake organizations).

You need to test-drive this.